supervisor/api/addons.py

```diff
@@ -137,10 +137,10 @@ def _extract_addon(
 
         addon = self.sys_addons.get(addon_slug)
         if not addon:
-            raise APIError("Addon does not exist")
+            raise APIError(f"Addon {addon_slug} does not exist", addon_slug)
```

Comment:
The second argument is a mistake, right?

Reply:
Yes, it is!

Adjusting...